### DISCUSSION TITLE
Remove site from github pages after repository transferred

### DISCUSSION THREAD

**Discussion by aramanenka** [community/community#78129] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

1. I set up github pages for my repository. It worked perfect. 
2. Then I transferred that repository to another account and new owner updated github pages settings. 
3. But I my old site is still available (and what is critical it is available for indexing)

Does someone know how to remove site from github pages if associated repository was transferred to another account?
thanks.

**Comment by naveenalla3000** [comment]
Ask the New Owner to go account and pages and then ask the New Owner to disable site.

**Reply by aramanenka** [reply]
Thanks for the answer, but:
new owner has their own github pages (they have another URL) and they work perfect. I suspect, they can remove only that working site, not mine.

**Comment by Gecko51** [comment]
Once the repo is transferred, the Pages site belongs to the new owner. The old URL `https://<olduser>.github.io/<repo>` either 301-redirects to `https://<newowner>.github.io/<repo>`, or it stops resolving, depending on how GitHub handled the transfer. You don't have a switch on your side to turn it off anymore because you're no longer the repo admin.

A few things you can actually do:

- **Ask the new owner to unpublish the site.** In their repo: `Settings` → `Pages` → on the right side there's an `Unpublish site` button (if Pages is enabled), or they can set `Source` to `None`. That kills the URL entirely.
- **If you still have admin on the transferred repo** (rare after a transfer), you can do the same from your end. Worth checking your access level, the transfer sometimes leaves you as a collaborator.
- **Get the new owner to delete the whole repo** if they're not using it. That removes the Pages site in the same move.

For the indexing side (which is what usually matters most):

- If you own the old URL in Google Search Console (you registered `<olduser>.github.io` as a property), you can file a **temporary removal request** there to drop it from results fast, then let the 301/404 do its job over time.
- Add a `<meta name="robots" content="noindex">` tag to the site's pages, but this only works if you can still push to the repo, which circles back to the new owner.
- For a full hard reset, the new owner can add a `robots.txt` at the root that disallows everything, then request re-crawl.

If the new owner is unresponsive, GitHub support can step in under the content removal policy if there's a real reason (DMCA, personal info, etc.), but "I just don't want it live" usually isn't enough to trigger that.

What's on the site that you need gone? Knowing that might open up a faster path (e.g. DMCA if it's your code/content, GDPR if it has personal info).
